histomicsui/web_client/views/body/ImageView.js

```diff
@@ -1103,6 +1103,48 @@ var ImageView = View.extend({
             case 's':
                 this.annotationSelector.selectAnnotationByRegion();
                 break;
+            case 'n':
+                if (!this.annotationSelector._activeAnnotation) {
+                    // If there's no active annotation, open the dialog to create a new one
+                    this.annotationSelector.createAnnotation();
+                } else if (this.drawWidget._drawingType) {
```

Comment:
I find this a little peculiar.  I would think that for all shapes, if the current shape is being drawn, selecting the letter would deselect the shape.  I think each drawing type would be of the form:
```
    if (this.annotationSelector._activeAnnotation) {
        if (this.drawWidget._drawingType === 'point') {
             this.drawWidget.cancelDrawMode();
        } else {
            this.drawWidget.drawElement(undefined, 'point');
        }
    }
```

In actual use, I think `l` would be better mapped to line rather than ellipse (and then `i` would map to ellipse?).  I think that `n` probably shouldn't map to drawing a shape.

Reply:
Makes sense to me, I've updated it with those changes :+1: